jbig2dec.yaml

```diff
@@ -21,8 +21,8 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      expected-sha256: 279476695b38f04939aa59d041be56f6bade3422003a406a85e9792c27118a37
-      uri: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9531/jbig2dec-${{package.version}}.tar.gz
+      expected-sha256: 7b63ff6470289547e7a3a0f145cb8ea6c2afffdd65645b7d87d3b7febc96fb3a
```

Comment:
[Non-blocking] But unless there is a reason for not using Git checkout, we should probably consider switching over. It looks like [the git repo cuts release tags](https://github.com/ArtifexSoftware/jbig2dec/tags), and it'd mean no issues with changing download URLs